easybuild/easyconfigs/l/LIBSVM/LIBSVM-3.24-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'LIBSVM'
+version = '3.24'
```

Comment:
@stefan-wolfsheimer We already have an easyconfig for LIBSVM 3.25 (cfr. #13472), any reason you're sticking to this older version?